Carbon/navigation/screens/Api.js

```diff
@@ -0,0 +1,5 @@
+// Testing Server
+//export const API_URL = "http://192.168.0.150:3000/api/"
+
+// Deployed Server
+export const API_URL = "http://Carbonserver-env.eba-pmpdtmpe.us-east-1.elasticbeanstalk.com/api/"
```

Comment:
Good work for helping us deploy the backend server, my only question is, is this a security risk having this link in the code itself?

Reply:
If it is we should fix that before deploying to app stores.

Reply:
that's the public facing link to the server, its just a url it doesnt have any keys or credentials